net/munge/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        dun munge 0.5.12 munge-
+categories          net security
+license             GPL-3
+maintainers         kornel.us:karl
```

Comment:
Please decide whether you want the port to be openmaintainer as explained in https://guide.macports.org/#project.update-policies.nonmaintainer. Add `openmaintainer` to the maintainers field if you want to mark it openmaintainer.